src/migrations/02-create-users.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     roleId: {
       type: Sequelize.INTEGER,
       allowNull: false,
-      defaultValue: 1
+      defaultValue: 1,
```

Comment:
The PR looks good @kingisaac95. However, I will like to understand why you prefer to add a comma on the last key of the JSON Object, as it is sometimes recommended not to.

```suggestion
      defaultValue: 1
```

Reply:
I use whichever standard my team requires. For most standard eslint configs, you need to have a trailing comma in objects and expanded method parameters.

The issues mentioned in the article you shared with _I think @ah-med_ is valid when you need to parse the object to a JSON. In this case, it doesn't apply @shegsteham

Reply:
Noted. Thanks for the clarification.